backend/open_webui/test/apps/webui/routers/test_retrieval.py

```
@@ -155,6 +160,28 @@ def fake_embedding_function(texts, prefix="", user=None):
+
+sys.modules["open_webui.services.uploads"] = SimpleNamespace(
+    UploadStage=SimpleNamespace(
+        UPLOAD_RECEIVED="upload_received",
+        CONTENT_EXTRACTED="content_extracted",
+        CHUNKS_CREATED="chunks_created",
+        EMBEDDINGS_GENERATED="embeddings_generated",
+        VECTORS_PERSISTED="vectors_persisted",
+        COMPLETED="completed",
+    ),
+    UploadProgress=DummyUploadProgress,
+    publish_upload_status=lambda *args, **kwargs: None,
```

Ruff (0.13.1)

Unused lambda argument: `args`

(ARG005)